benchmark/infrastructure/BenchmarkMeasurementContainer.h

```diff
@@ -228,7 +228,7 @@ class ResultTable : public BenchmarkMetadataGetter {
   @param row, column Which table entry to read. Starts with `(0,0)`.
   */
   template <typename T>
-  requires ad_utility::isTypeContainedIn<T, EntryType>
+  requires ad_utility::SameAsAnyTypeIn<T, EntryType>
```

Comment:
You switch between the `template <Concept T>` and `template <typename T> requires Concept<T>` syntax for no reason. 
If possible (that is the condition only depens on asingle parameter, I would consistently use the first syntax.
`template <SameAsAnyTypeIn<EntryType> T>`

Reply:
Done, see [4cf99e6](https://github.com/ad-freiburg/qlever/pull/1214/commits/4cf99e666a7a0141ee086a15e30dbb2743e14f65).